Dockerfile-build

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.6
+
+RUN apt-get update && apt-get install -y libltdl-dev ruby ruby-dev build-essential rpm && \
+    gem install fpm && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+
+COPY . /go/src/github.com/letsencrypt/boulder
+RUN mkdir /tmp/boulder-build
+WORKDIR /go/src/github.com/letsencrypt/boulder
+CMD [ "make", "rpm" ]
```

Comment:
this can be a RUN instead then in the script you can docker build... docker run tar ... | untar
example: https://github.com/jfrazelle/apparmor-docs/blob/master/Makefile#L9

just a different route to take but both are the same :)